test/test.clients.ts

```diff
@@ -131,10 +133,10 @@ describe('Clients', function () {
   it('should support default params', function (done) {
     var google = new googleapis.GoogleApis();
     var datastore = google.datastore({
-      version: 'v1beta2',
+      version: 'v1beta3',
```

Comment:
It feels like this change should be separated from the move to typescript but the owners of the repository would probably be the right people to comment on whether this is ok.